web/templates/settings/notification_settings.hbs

```diff
@@ -147,6 +147,13 @@
             </div>
         </div>
 
+        <div class="input-group">
+            <label for="realm_name_in_notifications_policy" class="dropdown-title">{{ settings_label.realm_name_in_notifications_policy }}</label>
+            <select name="realm_name_in_notifications_policy" class="setting_realm_name_in_notifications_policy prop-element settings_select bootstrap-focus-style" id="{{prefix}}realm_name_in_notifications_policy" data-setting-widget-type="number">
+                {{> dropdown_options_widget option_values=realm_name_in_notifications_policy_values}}
+            </select>
+        </div>
```

Comment:
We probably will want a `help_link_widget` pointing to the associated /help/ page, since it's probably not super intuitive what "Automatic" means.

@alya FYI.